### PR TITLE
OS6 Config Example

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

**Comment by NoSloppy** [comment]
Hey. By request, I'm commenting here. I thought an example, modern config sample for OS6 was a good idea too, and have an open PR for it.
Same idea here, I just also had POV included since I've been working on that.
While POV certainly doesn't need to be included, I think the header info is useful, similar to current example configs, explaining the hardware used so the settings can be understood better.
This is a copy/paste from that which maybe something similar should be added here?
```
/*
This is a sample configuration file for use with OS6 and above.
This saber has:
 - Proffieboard V2.2.
 - Two buttons: one POW, one AUX.
 - A 144 pixel blade on LED 2&3 and data 1 pads.
Examples given for:
- POV data file define
- Custom prop file inclusion (Fett263) with suggested minimum of #defines.
- OS6 Edit Mode compliant blade style
- POV blade style

More info on Edit Mode can be found here: 
https://fett263.s3.us-east-2.amazonaws.com/proffieOS6-edit-mode.html
More info on using POV can be found here:
https://github.com/profezzorn/ProffieOS/wiki/POV-(Persistance-Of-Vision)
If you have a saber similar to this one, make a copy and use the copy.
*/
```

**Comment by fdarosa2663** [comment]
Pretty sure anything you're working on would be OS7 since OS6 is already released.  Also, POV takes up way too much memory in my experience to recommend users try to use with Edit Mode and all of the other features in my prop.  If/when the POV stuff is part of the release I'd say show it in POV config as I'd expect users that want that would pretty much just have it as the main use, whereas this config is meant to be a common use by most users getting started with OS6.

**Comment by NoSloppy** [comment]
> Pretty sure anything you're working on would be OS7 since OS6 is already released.

Yes, comments open with " says OS6 and above" 

> If/when the POV stuff is part of the release I'd say show it in POV config as I'd expect users that want that would pretty much just have it as the main use, whereas this config is meant to be a common use by most users getting started with OS6.

Yes, agreed. As I said, the POV stuff should be removed.
My only suggestion was the descriptive comment section up top.

File: config/OS6_config_example.h [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+*/
+StylePtr<Layers<
+  ColorSelect<IntArg<STYLE_OPTION_ARG,0>,TrInstant,StripesX<Int<6000>,Scale<SlowNoise<Int<2000>>,Int<-1600>,Int<-3200>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,RandomPerLEDFlicker<Mix<Int<10280>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,Mix<Int<1285>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,BrownNoiseFlicker<Mix<Int<1285>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,Mix<Int<16448>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,300>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,RandomPerLEDFlicker<Black,Mix<Int<16448>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,StripesX<Int<1500>,Scale<SlowNoise<Int<2500>>,Int<-3000>,Int<-5000>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,Mix<Int<10280>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,Mix<Int<2570>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,
+  ColorSelect<IntArg<SWING_OPTION_ARG,0>,TrInstant,AlphaL<Black,Int<0>>,
+  TransitionEffectL<TrConcat<TrExtend<30000,TrFade<300>>,Pulsing<RgbArg<SWING_COLOR_ARG,Rgb<255,255,255>>,Mix<Int<16384>,Black,RgbArg<SWING_COLOR_ARG,Rgb<255,255,255>>>,3000>,TrFade<300>>,EFFECT_FORCE>>,
```
Comment (profezzorn) [inline config/OS6_config_example.h:72]: I think people are going to have a hard time seeing where the style ends if you have blank lines inside the styles.

Comment (profezzorn) [reply config/OS6_config_example.h:72]: Or is the blank styles just github being weird?

Comment (profezzorn) [reply config/OS6_config_example.h:72]: Is there any indentation here, I can't tell...

**Comment by fdarosa2663** [comment]
Top comment added per NoSloppy suggestion

File: config/OS6_config_example.h (hunk 1 above)
Comment (fdarosa2663) [reply config/OS6_config_example.h:88]: I don't see blanks and it's indented on my end, maybe github is acting up?

Comment (NoSloppy) [reply config/OS6_config_example.h:88]: looks ok on my end.

**Pull request merged**